engine.py

```diff
@@ -85,10 +85,25 @@ def get_bishop_moves(self, r, c, moves):
 
 			return parameter(s):
 			None
- 		"""
+		"""
 
- 		##TODO
- 		pass
+		##TODO
+		if self.light_to_move:
+			direction = [(-1,-1), (1,1),(1,-1), (-1,1)] # possible direction
```

Comment:
Change _direction_ from list of tuples to a tuple of tuples, since you don't need to edit it in runtime. And make spacing between elements uniform. i.e 
((-1,-1),(1,1),(1,-1),(-1,1))  : no spacing
or
((-1,-1), (1,1), (1,-1), (-1,1)) : single space